ci-scripts/yocto2.groovy

```diff
@@ -240,7 +248,9 @@ void buildManifest(String variantName, String imageName, String layerToReplace="
         boolean smokeTests = getBoolEnvVar("SMOKE_TEST", false)
         boolean bitbakeTests = getBoolEnvVar("BITBAKE_TEST", false)
         boolean yoctoCompatTest = getBoolEnvVar("YOCTO_COMPATIBILITY_TEST", false)
-        setupBitbake(yoctoDir, templateConf, doArchiveCache, smokeTests, analyzeImage)
+        boolean freescaleEULA = getBoolEnvVar("ACCEPT_FREESCALE_EULA", false)
+
+        setupBitbake(yoctoDir, templateConf, doArchiveCache, smokeTests, analyzeImage, freescaleEULA)
```

Comment:
Just call getBoolEnvVar() in setupBitbake() instead of passing yet another argument?